modelina-website/src/components/playground/options/GeneralOptions.tsx

```diff
@@ -70,10 +70,28 @@ class GeneralOptions extends React.Component<
         <h3 className="text-lg font-medium leading-6 text-gray-900">
           General options
         </h3>
-        <li>
+        <li className="relative">
           <label className="flex items-center py-2 justify-between cursor-pointer">
-            <span className="mt-1 max-w-2xl text-sm text-gray-500">
+            <span className="mt-1 max-w-2xl text-sm text-gray-500 relative group flex items-center">
+              <svg
+                className=" -ml-5 mr-2"
+                width="7"
+                height="13"
+                viewBox="0 0 7 13"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  d="M5.25 0C4.2875 0 3.5 0.73125 3.5 1.625C3.5 2.51875 4.2875 3.25 5.25 3.25C6.2125 3.25 7 2.51875 7 1.625C7 0.73125 6.2125 0 5.25 0ZM2.625 4.0625C1.1725 4.0625 0 5.15125 0 6.5H1.75C1.75 6.045 2.135 5.6875 2.625 5.6875C3.115 5.6875 3.5 6.045 3.5 6.5C3.5 6.955 1.75 9.165 1.75 10.5625C1.75 11.96 2.9225 13 4.375 13C5.8275 13 7 11.9112 7 10.5625H5.25C5.25 11.0175 4.865 11.375 4.375 11.375C3.885 11.375 3.5 11.0175 3.5 10.5625C3.5 9.9775 5.25 7.5725 5.25 6.5C5.25 5.18375 4.0775 4.0625 2.625 4.0625Z"
+                  fill="#DBD9D9"
+                />
+              </svg>
               Output type
+              <span className="opacity-0 invisible group-hover:opacity-100 group-hover:visible transition-opacity duration-300 ease-in ">
+                <span className="absolute -left-1/2 bg-blue-200 text-white text-xs py-1 px-2 rounded-lg bottom-full transform translate-x-[-7rem] translate-y-1/2">
+                  Additional information about the output type
```

Comment:
Whats the size limitation on this?

Reply:
> What happens if we want to link to something in the description? thinking

sure links can be added here :partying_face: 

> Can this setup handle multiple groups in the same document?

yes we this setup can be made scalable :+1: 

> Whats the size limitations on this?

there are no size limitations on this but i am thinking that we should keep a size limitation on the width and add a scroll option to scale it horizontally after a maximum height is reached, :smile:

Reply:
https://github.com/asyncapi/modelina/assets/65810424/1d34703e-08e8-4c57-be4b-c86008f9f38f

Reply:
i think the video got bit glitchy so here are the screenshots

Reply:
![Screenshot from 2023-06-03 06-39-23](https://github.com/asyncapi/modelina/assets/65810424/13b997f6-0b5b-4613-9e44-0c59061c311c)

Reply:
But if you try to click the link the box disappear right?

Reply:
@jonaslagoni i have fixed that !! :smile_cat: 
https://deploy-preview-1366--modelina.netlify.app/playground

Reply:
What about mobile view? It seems to be unusable there 🤔